src/calculateRentalCost.js

```diff
@@ -28,7 +28,27 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
+  const rentCostPerDay = 40;
+
+  // if (days <= 0) {
+  //   return 'Error';
+  // } else if (days.isFinite !== true) {
+  //   return 'Error';
+  // } else
+  if (days > 0 && days < 3) {
+    result = rentCostPerDay * days;
+
+    return result;
+  } else if (days >= 3 && days < 7) {
```

Comment:
```suggestion
  } else if (days < 7) {
```
Якщо ти в кожній перевірці пишеш return, то можна використовувати просто if, а не else if
Крім того, можна прибрати перевірку days >= 3, адже якщо не пройшла перша перевірка, то днів точно не менше трьох